src/platform/nxp/k32w/k32w1/BUILD.gn

```diff
@@ -58,13 +58,20 @@ static_library("k32w1") {
     "ram_storage.h",
   ]
 
+  public = [
+    "${chip_root}/src/app/reporting/reporting.h",
```

Comment:
public on static libraries `Declare public header files for a target` according to `gn help public`.

however here src/app/reporting/reporting.h does not belong to k32w platform, so it seems very odd for the platform to declare "this is one of my public headers". What is going on here?

Reply:
It's deprecated. I forgot to remove this.